packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/InputGroup/components/QuestionInput.spec.js

```diff
@@ -0,0 +1,259 @@
+import { expect } from 'chai'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { QuestionInput } from './QuestionInput'
+
+describe('Component > QuestionInput', function () {
+  describe('with checkbox type', function () {
+    it('should render without crashing', function () {
+      render(
+        <Grommet
+          theme={zooTheme}
+          themeMode='light'
+        >
+          <QuestionInput
+            option={{
+              label: 'Eating',
+              value: 'TNG'
+            }}
+            questionId='WHTBHVRSDS'
+            theme={zooTheme}
+            type='checkbox'
+          />
+        </Grommet>
+      )
+      expect(screen).to.be.ok()
+    })
+
+    it('should render the label', function () {
+      render(
+        <Grommet
+          theme={zooTheme}
+          themeMode='light'
+        >
+          <QuestionInput
+            option={{
+              label: 'Eating',
+              value: 'TNG'
+            }}
+            questionId='WHTBHVRSDS'
+            theme={zooTheme}
+            type='checkbox'
+          />
+        </Grommet>
+      )
+      expect(screen.getByText('Eating')).to.exist()
+    })
+
+    it('should have input not as the document active element', function () {
```

Comment:
This is a little hard to understand. Maybe `it('should not be focused')` or `it('should not be the document active element')`?

Reply:
Oof yes that was not a clear test description, fixed 👍